doc/asciidoc/pagerank.adoc

```diff
@@ -1,107 +1,109 @@
-* we assume that a page A has pages T1...Tn which point to it (i.e., are citations).
-* d is a damping factor which can be set between 0 and 1. It is usually set to 0.85.
-* `C(A)` is defined as the number of links going out of page A.
+* we assume that a page `A` has pages `T1` to `Tn` which point to it (i.e., are citations).
+* `d` is a damping factor which can be set between 0 and 1.
+It is usually set to 0.85.
+* `C(A)` is defined as the number of links going out of page `A`.
 
-The PageRanks form a probability distribution over web pages, which means that the sum of all web pages' PageRanks will be one.
+The PageRanks form a probability distribution over web pages, which means that the sum of all web pages' PageRanks will be 1.
 // end::formula[]
 
-== When to use it / use-cases
+== Use-cases - when to use the PageRank algorithm
 
 // tag::use-case[]
 PageRank can be applied across a wide range of domains.
-These are some notable use cases:
+The following are some notable use-cases:
 
 * Personalized PageRank is used by Twitter to present users with recommendations of other accounts that they may wish to follow.
-The algorithm is run over a graph containing shared interests and common connections.
+The algorithm is run over a graph which contains shared interests and common connections.
 Their approach is described in more detail in https://web.stanford.edu/~rezab/papers/wtf_overview.pdf[WTF: The Who to Follow Service at Twitter^].
 
-* PageRank has been used to rank public spaces or streets to predict traffic flow and human movement in these areas.
-The algorithm is run over a graph of intersections connected by roads and the PageRank score reflects the tendency of people to park, or end their journey, on each street.
+* PageRank has been used to rank public spaces or streets, predicting traffic flow and human movement in these areas.
+The algorithm is run over a graph which contains  intersections connected by roads, where the PageRank score reflects the tendency of people to park, or end their journey, on each street.
 This is described in more detail in https://arxiv.org/pdf/0804.1630.pdf[Self-organized Natural Roads for Predicting Traffic Flow: A Sensitivity Study^].
 
 * PageRank can be used as part of an anomaly or fraud detection system in the healthcare and insurance industries.
-It can help find doctors or providers that are behaving in an unusual manner and then feed the score into a machine learning algorithm.
+It can help find doctors or providers that are behaving in an unusual manner, and then feed the score into a machine learning algorithm.
 
-There are many more use cases which you can read about in David Gleich's https://arxiv.org/pdf/1407.5107.pdf[PageRank beyond the web^]
+There are many more use cases, which you can read about in David Gleich's https://arxiv.org/pdf/1407.5107.pdf[PageRank beyond the web^]
 // end::use-case[]
 
-== Constraints / when not to use it
+== Constraints - when not to use the PageRank algorithm
```

Comment:
did you add the algorithm name for anchors? b/c it's already in the page-rank section?
so perhaps more useful to add an explicit anchor-id ?